roles/openshift_master_facts/tasks/main.yml

```diff
@@ -0,0 +1,145 @@
+---
+- name: Check for RPM generated config marker file .config_managed
+  stat:
+    path: /etc/origin/.config_managed
+  register: rpmgenerated_config
+
+- name: Remove RPM generated config files if present
+  file:
+    path: "/etc/origin/{{ item }}"
+    state: absent
+  when: rpmgenerated_config.stat.exists == true and deployment_type in ['openshift-enterprise', 'atomic-enterprise']
```

Comment:
I'm wondering if 'online' should be in this list as well, since it is using the same binary packaging and could potentially have the generated configs.